package.json

```diff
@@ -19,11 +19,15 @@
   },
   "scripts": {
     "precommit": "lint-staged",
+    "prepublish": "grunt peerDepInstall",
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
     "test": "grunt test"
   },
+  "peerDependencies": {
+    "@dojo/cli": "file:./tarballs/dojo-cli-3.0.0-beta.1.tgz",
```

Comment:
same question re peer deps - we should get away from `prepublish` anyway as it's deprecated.

Reply:
It's only a peer dependency for the time being so this can be made into a tarball and installed itself.